docs/documentation/case.md

```diff
@@ -236,6 +236,11 @@ and use `patch_icpp(i)%%geometry = 7` and `patch_icpp(i)%%hcid = 200` in the inp
 Additional variables can be declared in `Hardcoded1[2,3]DVariables` and used in `hardcoded1[2,3]D`.
 As a convention, any hard coded patches that are part of the MFC master branch should be identified as 1[2,3]xx where the first digit indicates the number of dimensions.
 
+Also Several custom hard-coded patches are already defined to support common workflows. These include:
```

Comment:
what's going on here. These include: 

but then there are no lines below it with a list....